docs/sdk/v3/guides/liquidity/01-minting-a-position.md

```diff
@@ -3,4 +3,90 @@ id: minting
 title: Minting a Position
 ---
 
-TODO
\ No newline at end of file
+## Introduction
+
+This guide will cover how to create (or mint) a liquidity position on the Uniswap V3 protocol.
+It is based on the [Minting a position code example](https://github.com/Uniswap/examples/tree/main/v3-sdk/minting-position), found in the Uniswap code examples [repository](https://github.com/Uniswap/examples).
+To run this example, check out the examples's [README](https://github.com/Uniswap/examples/blob/main/v3-sdk/quoting/README.md) and follow the setup instructions.
+
+:::info
+
+For a briefer on the SDK and to learn more about how these guides connect to the examples repository, please visit our [background coverage](./01-background.md) page!
+
+:::
+
+In this example we will use  the `NonfungiblePositionManager` class to help us mint a liquidity position for the  **USDC - DAI** pair.
+The inputs are the **token in**, the **token out**,  the **amount in**, the **fee** and the **liquidity**.
+
+The guide will **cover**:
+1. Giving approval to the `NonfungiblePositionManager` contract to transfer our tokens.
+2. Fetching the Pool's constants and current state.
+3. Creating an instance of a `Position` class and using `addCallParameters` on our `NonfungiblePositionManager` to get the data for the transaction.
+
+At the end of the guide, given the inputs above with, we should be able to mint a liquidity position with the press of a button and view the position's id on the web application.
```

Comment:
Should there be a fourth step for actually sending the transaction?

Reply:
I think that's more of an implementation detail that we also skipped in the *quoting* example